mypy_django_plugin/transformers/models.py

```diff
@@ -390,3 +393,59 @@ def set_auth_user_model_boolean_fields(ctx: AttributeContext, django_context: Dj
     boolinfo = helpers.lookup_class_typeinfo(helpers.get_typechecker_api(ctx), bool)
     assert boolinfo is not None
     return Instance(boolinfo, [])
+
+
+def handle_annotated_type(ctx: AnalyzeTypeContext, django_context: DjangoContext) -> MypyType:
+    args = ctx.type.args
+    type_arg = ctx.api.analyze_type(args[0])
+    api = cast(SemanticAnalyzer, ctx.api.api)  # type: ignore
+
+    if not isinstance(type_arg, Instance):
+        return ctx.api.analyze_type(ctx.type)
+
+    return get_or_create_annotated_type(api, type_arg)
+
+
+def get_or_create_annotated_type(
+    api: Union[SemanticAnalyzer, CheckerPluginInterface], model_type: Instance
```

Comment:
`api: SemanticAnalyzer`?

Reply:
Or even `TypeAnalyzer`

Reply:
It's unfortunately used both in `MethodContext` (`CheckerPluginInterface`) and `AnalyzeTypeContext` (where I can grab `SemanticAnalyzer` from the `api.api` attribute).